plugins/tiddlywiki/highlight-legacy/howto.tid

```diff
@@ -0,0 +1,10 @@
+title: $:/plugins/tiddlywiki/highlight/howto
+
+! Supporting Additional Languages
+ 
+The [[highlight.js|https://github.com/highlightjs/highlight.js]] project supports many languages. Only a subset of these languages are supported by the plugin. It is possible for users to change the set of languages supported by the plugin by following these steps:
+ 
+# Go to the highlight.js project [[download page|https://highlightjs.org/download/]], select the language definitions to include, and press the Download button to download a zip archive containing customised support files for a highlight.js syntax highlighting server.
+# Locate the `highlight.pack.js` file in the highlight plugin -- on a stock Debian 8 system running Tiddlywiki5 under node-js it is located at `/usr/local/lib/node_modules/tiddlywiki/plugins/tiddlywiki/highlight/files/highlight.pack.js`.
+# Replace the plugin `highlight.pack.js` file located in step 2 with the one from the downloaded archive obtained in step 1.
+# Restart the Tiddlywiki server.
```

Comment:
It would be great if we could write instructions for users who only use TiddlyWiki in the browser.

Reply:
@Jermolene 

Just want to make sure we are talking about the legacy version. I didn't plan to touch its code base so the instructions stay unchanged. Is it worth the effort to backport browser-based method of adding languages? Users would also need to hunt down v9 language files that will be hard to locate as time goes by.

Reply:
Of course, I'd missed that, thank you @cdruan